src/Uno.UI/ts/Windows/Storage/ApplicationDataContainer.ts

```diff
@@ -19,10 +19,15 @@ namespace Windows.Storage {
 
 			const storageKey = ApplicationDataContainer.buildStorageKey(params.Locality, params.Key);
 
-			if (localStorage.hasOwnProperty(storageKey)) {
-				ret.HasValue = true;
-				ret.Value = localStorage.getItem(storageKey);
-			} else {
+			try {
+				if (localStorage.hasOwnProperty(storageKey)) {
+					ret.HasValue = true;
+					ret.Value = localStorage.getItem(storageKey);
+				} else {
+					ret.Value = "";
+					ret.HasValue = false;
+				}
+			} catch (e) {
```

Comment:
nit: It seems you don't need the variable

```suggestion
			} catch {
```